### ISSUE TITLE
"watch" task doesn't detect changes in index.html

### ISSUE THREAD

**Issue by eiriarte** [yeoman/generator-webapp_DEPRECATED#230]
Changes in .css/.scss files are detected and the webpage is reloaded as expected.

But when I change index.html the "watch" task does nothing.

[**Line 65**](https://github.com/yeoman/generator-webapp/blob/master/app/templates/Gruntfile.js#L65) in **app/templates/Gruntfile.js** says:

```
'<%%= yeoman.app %>/{,*/}.html',
```

Looks like it should be:

```
'<%%= yeoman.app %>/{,*/}*.html',
```

(I made a similar change in my local Gruntfile.js and now it works fine again).

I hope that helps. This is my first Github issue, any advice is appreciated! :)

Cheers!

**Comment by addyosmani** [comment]
@eiriarte could you try our 0.4.6 release (`npm update -g generator-webapp`) and let us know if the issue is fixed for you now?

**Comment by eiriarte** [comment]
@addyosmani Yes, fixed, with 0.4.6 it works again.

Changes in .html files are detected and the page is nicely reloaded in the browser.

I also tried to create, delete and change .scss and .js files, and it also works.

Thank you guys for all the work you put in these awesome tools!
